trl/trainer/dpo_trainer.py

```diff
@@ -364,6 +364,8 @@ def dpo_loss(
             losses = -F.logsigmoid(self.beta * logits)
         elif self.loss_type == "hinge":
             losses = torch.relu(1 - self.beta * logits)
+         else:
+            raise ValueError(f"Unknown loss type: {self.loss_type}")
```

Comment:
Small nit to make it more user-friendly :) 

```suggestion
            raise ValueError(f"Unknown loss type: {self.loss_type}. Should be one of ['sigmoid', 'hinge']")
```

Reply:
btw i think the indent is off